src/cli/commands/execute/mod.rs

```diff
@@ -72,7 +72,7 @@ impl CommandExecution<ExecuteOutput> for ExecuteArgs {
 		let execution_uuid = Uuid::new_v4();
 		init_buffer(execution_uuid);
 		// Run the main function of cairo contract
-		let (mut runner, mut vm) = cairo_run(
+		let (mut runner, mut vm) = cairo_pre_run(
```

Comment:
You can get rid of the whole `execute` command. It causes trouble for code maintainability and is not a support we want to have in the long run.
I will create an issue for this and assign it to you.

Maybe start with this one, there rebase your current branch on it

Reply:
Ok if you prefer, you first merge those. Then get rid of the `execute` command.
You tell me